### PR TITLE
adding akka diagnostics and some cleansing

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
References https://github.com/akka/akka-diagnostics/issues/19

Kept in draft until akka-diagnostics 2.0.0-M3 release

### CODE REVIEW

File: akka-sample-cluster-client-grpc-scala/build.sbt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
 import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
 
-val akkaVersion = "2.7.0"
+val AkkaVersion = "2.7.0"
+val AkkaDiagnosticsVersion = "2.0.0-M3"
+val ScalaTestVersion = "3.0.8"
```
Comment (patriknw) [inline akka-sample-cluster-client-grpc-scala/build.sbt:6]: this was 3.1.1, why moving backwards?

**Review by patriknw — COMMENTED** [review]
looking good, feel free to cut the diagnostics M3 to unlock these sample PRs

Did you try running some of them?

File: akka-sample-cluster-client-grpc-scala/build.sbt (hunk 1 above)
Comment (franciscolopezsancho) [reply akka-sample-cluster-client-grpc-scala/build.sbt]: fixed

**Comment by franciscolopezsancho** [comment]
I did not run any of them. I was waiting to release the M3. I was wondering if only run a few or run them all. I was more inclined toward the latter.

File: akka-sample-cluster-java/build.sbt [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
 import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
 
-val akkaVersion = "2.7.0"
+val AkkaVersion = "2.7.0"
+val AkkaDiagnosticsVersion = "2.0.0-M3"
+val LogbackClassicVersion = "1.2.11" 
+val ScalaTestVersion = "3.0.8"
```
Comment (patriknw) [inline akka-sample-cluster-java/build.sbt:7]: same thing in several places

Comment (franciscolopezsancho) [reply akka-sample-cluster-java/build.sbt:7]: `"3.0.8"` was the previous version in the build.sbt (in most build.sbt). Are you suggesting bumping it to `3.1.0` in all?

Comment (patriknw) [reply akka-sample-cluster-java/build.sbt:7]: ok, while you are anyway changing it you can bump to 3.1.0

Comment (franciscolopezsancho) [reply akka-sample-cluster-java/build.sbt:7]: There is one project with '3.1.1' I'll use that for the rest.

**Comment by franciscolopezsancho** [comment]
I checked it in quite a few projects. In a couple of them I found some unrelated issues that I raised in a separate [issue](https://github.com/akka/akka-samples/issues/275).

**Comment by franciscolopezsancho** [comment]
I checked it works in quite a few projects: all good. In a couple of them I found some unrelated issues that I raised in a separate [issue](https://github.com/akka/akka-samples/issues/275).

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**